project_task_stage_change_restriction/readme/CONFIGURE.md

```
@@ -0,0 +1,9 @@
+Go to "Project > Configuration > Task Stages" and select or create a new task stage.
+Configure the following fields in the "Stage Change Restriction" group:
+  -  Assigned Only. If enabled, only users assigned to the task can move it into this stage.
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)